src/rest-server/src/controllers/v2/storage.js

```diff
@@ -31,7 +32,11 @@ const getStorageServer = async (req, res, next) => {
 
 const getStorageServers = async (req, res, next) => {
   try {
-    const names = req.body.names;
+    const names = isEmpty(req.query.names)
```

Comment:
As the API doc says, the query names will always be an array. So if the query is a string instead of a one element array, it will be a bad request. 

Or the API doc should add an additional description that query accepts both array and string format.

Reply:
Http Get cannot guarantee the parameter is array. Adding multiple query example in DOC per discussion.